include/swift/Basic/SupplementaryOutputPaths.h

```diff
@@ -60,6 +60,11 @@ struct SupplementaryOutputPaths {
   /// It is currently only used with WMO, but could be generalized.
   std::string TBDPath;
 
+  /// The path to which we should emit a textual module interface.
```

Comment:
How about adding another sentence here, regarding the purpose of the textual module interface?

Reply:
Hm. I'd rather put that on the function that actually generates the thing. What do you think?

Reply:
Good question! Where are the explanations for the other supplementary outputs? How would one find the function that does the generation if one were looking at this declaration? Seems easier to go from the use of the declaration to the declaration than the reverse, because there is a plurality of uses, but only one declaration. I guess I lean towards the declaration, but any choice that makes it discoverable (e.g. cross-references) seems fine.